examples/with-thirdpartyemailpassword/google.js

```diff
@@ -0,0 +1,46 @@
+import { authorize } from 'react-native-app-auth';
+import SuperTokens from "supertokens-react-native";
+import { BASE_URL } from "./App";
+
+import axios from 'axios';
+
+// Adds request and response interceptors to the axios instance (Not required if using fetch)
+SuperTokens.addAxiosInterceptors(axios);
+
+/*
+    We use react-native-app-auth to login with Google and then use the response to sign in with SuperTokens
+*/
+const loginWithGoogle = async () => {
+    // Full configuration options here: https://github.com/FormidableLabs/react-native-app-auth#config
+    const config = {
+        issuer: 'https://accounts.google.com',
+        clientId: '1060725074195-c7mgk8p0h27c4428prfuo3lg7ould5o7.apps.googleusercontent.com',
```

Comment:
So this technically works with a different package name, the problem is that you still need to use the current redirect_url. For example lets say the user wants to change the package name to com.test, when they use the config

```js
const config = {
        issuer: 'https://accounts.google.com',
        clientId: '1060725074195-c7mgk8p0h27c4428prfuo3lg7ould5o7.apps.googleusercontent.com',
        redirectUrl: 'com.test:/oauthredirect',
        scopes: ["https://www.googleapis.com/auth/userinfo.email"]
    };
```

When you try to log in with Google, the google auth page gives an error for invalid redirect url. If you change the package name but then use the current redirect url `com.demoapp:/oauthredirect` then everything works fine and the user gets signed in.

Documenting this will be tricky and adds complication in terms of understanding it

Reply:
That's weird.. cause i did not input `com.demoapp:/oauthredirect` anywhere in the google dashboard... So why would it complain about `com.test:/oauthredirect`?

Reply:
@nkshah2 ^^